UPDATING_TO_2.0.md

```diff
@@ -0,0 +1,986 @@
+)
+
+port = k8s.V1ContainerPort(name='http', container_port=80)
+secret_file = Secret('volume', '/etc/sql_conn', 'airflow-secrets', 'sql_alchemy_conn')
+secret_env = Secret('env', 'SQL_CONN', 'airflow-secrets', 'sql_alchemy_conn')
+secret_all_keys = Secret('env', None, 'airflow-secrets-2')
+volume_mount = k8s.V1VolumeMount(
+    name='test-volume', mount_path='/root/mount_file', sub_path=None, read_only=True
+)
+
+k = KubernetesPodOperator(
+    namespace='default',
+    image="ubuntu:16.04",
+    cmds=["bash", "-cx"],
+    arguments=["echo", "10"],
+    labels={"foo": "bar"},
+    secrets=[secret_file, secret_env],
+    ports=[port],
+    volumes=[volume],
+    volume_mounts=[volume_mount],
+    name="airflow-test-pod",
+    task_id="task",
+    is_delete_operator_pod=True,
+    hostnetwork=False)
+```
+We decided to keep the Secret class as users seem to really like that simplifies the complexity of mounting
+Kubernetes secrets into workers.
+
+For a more detailed list of changes to the KubernetesPodOperator API, please read [here](#Changed-Parameters-for-the-KubernetesPodOperator)
+
+## Step 4: Update system configurations
+### Drop legacy UI in favor of FAB RBAC UI
+
+> WARNING: Breaking change
+
+Previously we were using two versions of UI, which were hard to maintain as we need to implement/update the same feature
+in both versions. With this release we've removed the older UI in favor of Flask App Builder RBAC UI. No need to set the
+RBAC UI explicitly in the configuration now as this is the only default UI. We did it to avoid
+the huge maintenance burden of two independent user interfaces
+
+Please note that that custom auth backends will need re-writing to target new FAB based UI.
+
+As part of this change, a few configuration items in `[webserver]` section are removed and no longer applicable,
+including `authenticate`, `filter_by_owner`, `owner_mode`, and `rbac`.
+
+Before upgrading to this release, we recommend activating the new FAB RBAC UI. For that, you should set
+the `rbac` options  in `[webserver]` in the `airflow.cfg` file to `true`
+
+```ini
+[webserver]
+rbac = true
+```
+
+In order to login to the interface, you need to create an administrator account.
+```
+airflow create_user \
+    --role Admin \
+    --username admin \
+    --firstname FIRST_NAME \
+    --lastname LAST_NAME \
+    --email EMAIL@example.org
+```
+
+If you have already installed Airflow 2.0, you can create a user with the command `airflow users create`.
+You don't need to make changes to the configuration file as the FAB RBAC UI is
+the only supported UI.
+```
+airflow users create \
+    --role Admin \
+    --username admin \
+    --firstname FIRST_NAME \
+    --lastname LAST_NAME \
+    --email EMAIL@example.org
+```
+
+### Breaking Change in OAuth
+
+The flask-ouathlib has been replaced with authlib because flask-outhlib has
+been deprecated in favour of authlib.
```

Comment:
Redundant sentence here -

> The flask-ouathlib has been replaced with authlib because flask-outhlib has
been deprecated in favour of authlib.
-->

> In Airflow 2.0, the `flask-outhlib` has been deprecated in favor of `authlib`.